collectors/ebpf.plugin/README.md

```diff
@@ -39,18 +39,104 @@ section for details.
 
 ## Charts
 
-The eBPF collector creates an **eBPF** menu in the Agent's dashboard along with three sub-menus: **File**, **VFS**, and
-**Process**. All the charts in this section update every second. The collector stores the actual value inside of its
-process, but charts only show the difference between the values collected in the previous and current seconds.
+The eBPF collector creates an **eBPF** menu in the Agent's dashboard along with two sub-menus: **Socket**, and
+**Process**. The plugin also creates charts on different menus, like System Overview, Memory, Disks, Filesystem, 
+Mount Points and Applications. All the charts in this section update every second.
+The collector stores the actual value inside of its process, but charts only show the difference between the values
+collected in the previous and current seconds.
 
-### File
+### System overview
 
-This group has two charts demonstrating how software interacts with the Linux kernel to open and close file descriptors.
+Not all charts within the System Overview menu are enabled by default, because they add around 100ns overhead for each 
+function call, this number is small for a human perspective, but the functions are called many times creating an impact 
+on host.
+See the [configuration](#configuration) section for details about how to enable them.
+
+#### Swap
+
+Inside the swap submenu the eBPF plugin creates the chart `swapcalls`; this chart is displaying when processes are calling
+functions `swap_readpage` and `swap_writepage`, which are functions responsible for doing IO in swap memory.
+
+### Memory
+
+In the memory submenu the eBPF plugin creates two submenus **page cache** and **synchronization** with the following
+organization:
+
+* Page Cache adds 
+  * Page cache ratio
+  * Dirty pages
+  * Page cache hits
+  * Page cache misses
+* Synchronization
+  * File sync
+  * Memory map sync
+  * File system sync
+  * File range sync
+
+#### Page cache
+
+Page cache is adding `kprobes` for the kernel functions `add_to_page_cache_lru`, `mark_page_accessed`,
+`account_page_dirtied`, and `mark_buffer_dirty`. Netdata monitors page cache access giving system-level and 
+application-level metrics.
+
+#### Page cache ratio
```

Comment:
"Page cache" is a submenu according to the Memory bulleted list, but "Page cache", "Page cache ratio", and "Dirty pages" have the same heading level (number of `#`).

It is not the only place you have that problem, please go through the "Charts" section and fix where is needed.

If i understand you right, you want to use:
- `###` top level menu section
- `####` sub-menu section
- `#####` a chart (title?)

Reply:
All right, I moved the data from `Page cache` to submenu and I removed the menu, because they were already explained in `Mem`. 

I removed `Synchronization` without to move content, because the `syscalls` monitored are already cited in chart description.